src/lib/updating-element.ts

```diff
@@ -266,14 +353,19 @@ export abstract class UpdatingElement extends HTMLElement {
   private static _propertyValueFromAttribute(value: string,
                                              options?: PropertyDeclaration) {
     const type = options && options.type;
-    if (type === undefined) {
+    let serializer = options && options.serializer;
+
+    if (type !== undefined) {
```

Comment:
This requires omitting `type` if you want to run a custom serializer.  Would it make more sense to instead drop in the a default serializer based on type only if you didn't explicitly provider one?